Marlin/src/pins/pins_BIGTREE_SKR_MINI_V1_1.h

```diff
@@ -0,0 +1,193 @@
+/**
+ * Marlin 3D Printer Firmware
+ * Copyright (C) 2016 MarlinFirmware [https://github.com/MarlinFirmware/Marlin]
+ *
+ * Based on Sprinter and grbl.
+ * Copyright (C) 2011 Camiel Gubbels / Erik van der Zalm
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef TARGET_STM32F1
+  #error "Oops! Select an STM32F1 board in 'Tools > Board.'"
+#endif
+
+#ifndef BOARD_NAME
+  #define BOARD_NAME "BIGTREE SKR mini V1.1"
+#endif
+
+// Ignore temp readings during develpment.
+//#define BOGUS_TEMPERATURE_FAILSAFE_OVERRIDE
+
+//
+// Limit Switches
+//
+#define U_MIN_PIN          -1
+#define V_MIN_PIN          -1
+#define W_MIN_PIN          -1
+#define X_STOP_PIN         PC2
+#define Y_STOP_PIN         PC1
+#define Z_STOP_PIN         PC0
```

Comment:
Is that Marlin 1.1 style ? The board has MIN and MAX (6 endstop so)

Reply:
> Is that Marlin 1.1 style ? The board has MIN and MAX (6 endstop so)
Well, the firmware has the Min and MAX you mentioned, and the basic configuration is the same.

Reply:
@tpruvot Well, the firmware has the Min and MAX you mentioned, and the basic configuration is the same.

Reply:
Marlin is happy with `X_STOP_PIN`, etc. It auto-assigns them to the `X_MIN_PIN` or `X_MAX_PIN` (etc.) based on homing direction set in the config.